Include/Acidanthera/Library/OcBootManagementLib.h

```diff
@@ -2075,6 +2078,40 @@ OcAddEntriesFromBootEntryProtocol (
   IN     BOOLEAN        CreateForHotKey
   );
 
+#pragma pack (push, 1)
+
+/**
+  Useful internal vars from firmware UI protocol (cMP 144.0.0.0.0 ROM).
+**/
+typedef PACKED struct {
+  BOOLEAN                         mGopAlreadyConnected;
+  EFI_GRAPHICS_OUTPUT_PROTOCOL    *mGop;
+  EFI_SYSTEM_TABLE                *gST;
+  EFI_BOOT_SERVICES               *gBS;
+  EFI_RUNTIME_SERVICES            *gRT;
+  EFI_DXE_SERVICES                *gDS;
+} APPLE_FIRMWARE_UI_VARS;
+
+/**
+  Represents the actual memory layout of the targeted firmware.
+**/
+typedef PACKED struct {
+  APPLE_FIRMWARE_USER_INTERFACE_PROTOCOL    UIProtocol;
+  APPLE_FIRMWARE_UI_VARS                    UIVars;
+} APPLE_FIRMWARE_UI_LAYOUT;
+#pragma pack (pop)
+
+/**
+  Force Apple Firmware UI to reconnect to current console GOP.
+
+  @retval EFI_SUCCESS   Picker was successfully unlocked.
+  @retval other         Compatible firmware UI protocol for unlock could not be found.
+**/
+EFI_STATUS
+OcUnlockAppleFirmwareUI (
```

Comment:
Maybe rename to OcReconnectFirmwareUI? There is no unlocking here.

Reply:
Hi - hmm, the thing about `OcReconnectAppleFirmwareUI` (now that I try renaming it, and looking at the code) is that it doesn't reconnect GOP at the time of the call, instead it 'unlocks' it, or opens it up again, so that it will reconnect, next time it's asked to. Hence previous name `OcReconnectAppleFirmwareUI`. Which also fits with my naming of the method in the firmware UI protocol interface, `ConnectGop`, which is the method which is affected by this.

Reply:
Well, ok, I guess it is fine to leave it as is then.